native/src/sagas/__tests__/loadCityContent.spec.ts

```diff
@@ -29,6 +29,7 @@ import loadPois from '../loadPois'
 
 jest.mock('../../utils/helpers', () => ({
   ...jest.requireActual('../../utils/helpers'),
+  log: jest.fn(),
   reportError: jest.fn()
 }))
```

Comment:
I think we should mock this by default or not mock this at all. Why are we mocking log and reportError?

Reply:
`reportError` needs to be mocked/spied to be able to assert on it. Removed mocking of `log`